docs/developer-guide.md

```diff
@@ -26,6 +26,30 @@ In this debug mode, `DotnetRunner` does not launch the .NET application, but wai
 
 Now you can run your .NET application with any debugger to debug your application.
 
+### Debugging UDF
+
+**Note that this is currently supported only on Windows.**
```

Comment:
For Linux, is [this](https://github.com/Microsoft/MIEngine/wiki/Offroad-Debugging-of-.NET-Core-on-Linux---OSX-from-Visual-Studio) going to be helpful?

Reply:
Probably not. `Debugger.Launch()` seems to work only for Windows: https://github.com/dotnet/corert/issues/7824

Reply:
That will work for driver side debugging.

Reply:
I wrote "This _might_ not even work with CoreCLR" - I'm not a CoreCLR debugger dev. The issue was opened in the Core**RT** repo - if you need an authoritative answer from a CoreCLR debugger dev, you might want to open an issue in the CoreCLR repo.